app/src/App.vue

```diff
@@ -47,16 +49,42 @@ onBeforeMount(async () => {
         Sentry.captureException(err);
     }
 });
+
+// Wait 1 second to allow the socket connection to be established before checking the connection status
+setTimeout(() => {
```

Comment:
I always dislike artificial timeouts like this. While it's a working solution, and I'm not necessarily advocating a different approach right now, it does trigger the thought in me if we can do something different. To me there's a few downsides with this approach:
- Someone might close the page or navigate away. This is the `App` component which will stay alive throughout the app, but in other components that means the `setTimeout` keeps floating around (that's why you generally want to do a [clearTimeout](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value) in a Vue component's `onBeforeDestroy`).
- Someone's connection might be slower, meaning that in this case they are perhaps not offline at all

One thing I'm thinking of is that we should track if the socket has tried to connect, and base the display of this banner on that (`hasTriedToConnect && !isConnected`). In my console when I disable the API I see a lot of `Failed to load resource: net::ERR_CONNECTION_REFUSED`, so the browser does know if you did at least one request or not.

Just paging @ivanslabbert to get his thoughts on this.